Sources/Sentry/Public/SentryDebugMeta.h

```diff
@@ -4,6 +4,13 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * This class is actually a DebugImage:
+ * https://develop.sentry.dev/sdk/event-payloads/debugmeta/#debug-images and should be renamed to
```

Comment:
If this was internal isn't it safe to rename now?

Reply:
This was already public see: https://github.com/getsentry/sentry-cocoa/blob/dd880f3da9884282cd0cde294af6118638b275cb/Sources/Sentry/Public/Sentry.h#L13.

The SentryDebugImageProvider was not public before.